tiledb/sm/config/config.h

```diff
@@ -179,6 +179,13 @@ class Config {
    */
   static const std::string SM_VACUUM_MODE;
 
+  /** The offset format to be used for variable-sized attributes. It can be one
```

Comment:
For doxygen-style block comments, we usually start the words on the line after `/**`, so:
```
/**
 * The offset ...
 * ...
 */
```